Makefile

```diff
@@ -0,0 +1,31 @@
+ifeq ($(TARGET),rhel)
+  DOCKERFILE := Dockerfile.rhel
```

Comment:
Your repository does not contain a Dockerfile.rhel, and neither does this PR. You need to create that.

Reply:
@rootAvish For testing DockerRHEL I added that in another PR https://github.com/fabric8-analytics/f8a-hpf-insights/pull/16

Reply:
It needs to be merged before #15, RHEL builds are not done for PRs.

Reply:
@rootAvish #16 is merged now.